gamemodes/terrortown/gamemode/shared/lang/de.lua

```diff
@@ -36,7 +36,7 @@ L.round_traitors_more = "Verräter, dies sind die Namen deiner Verbündeten: {na
 L.win_time = "Die Zeit ist abgelaufen. Die Verräter haben verloren."
 L.win_traitors = "Die Verräter haben gewonnen!"
 L.win_innocents = "Die Innos haben gewonnen!"
-L.win_bees = "Die Bienen haben gewonnen! (Es ist ein Unentschieden)"
+L.win_nones = "Die Bienen haben gewonnen! (Es ist ein Unentschieden)"
```

Comment:
```suggestion
L.win_nones = "Niemand hat gewonnen! (Es ist ein Unentschieden)"
```

Reply:
this is not important as is will be completely different in my PR